staging/src/k8s.io/client-go/federation/apis/federation/doc.go

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Looks like remotecommand imports federation types as well, can we remove the dependency on federation and kubeadm types?